packages/clients/meldemichel/API.md

```diff
@@ -1,20 +1,24 @@
 
 The method expects a single object with the following parameters.
 
 | fieldName       | type                                 | description                                                                                                                                                                                                                                                                             |
 | --------------- | ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | containerId     | string                               | ID of the container the map is supposed to render itself to.                                                                                                                                                                                                                            |
-| mode            | enum["COMPLETE", "SINGLE", "REPORT"] | See chapters below for an overview of the modes.                                                                                                                                                                                                                                        |
-| afmUrl          | string?                              | COMPLETE mode only. URL used in the AfM Button.                                                                                                                                                                                                                                         |
-| reportServiceId | string?                              | COMPLETE mode only. ID of the report layer to display.                                                                                                                                                                                                                                  |
-| configOverride  | object?                              | You will probably not need this object. It is included to override any internal configuration, should the need for quick reconfiguration arise. Please refer to the general documentation on what can be done here. If you need it, please communicate back with us what the issue was. |
+| mode            | enum["REPORT", "SINGLE", "COMPLETE"] | See chapters below for an overview of the modes.                                                                                                                                                                                                                                        |
```

Comment:
Why the reorder of the `enum`?

Reply:
Seemed appropriate to me to list the options relevant to users first. "COMPLETE" will probably only be used by us.